EarlySamples/InitialProject/InitialProject/Assets/Scripts/Test2/CameraFollow.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraFollow : MonoBehaviour
+{
+    public Transform target;
+
+    private Vector3 cameraOffset;
+
+    private float smoothFactor = 0.5f;
+
+    // Start is called before the first frame update
+    void Start()
+    {
+        cameraOffset = transform.position - target.transform.position;
+    }
+
+    // Update is called once per frame
+    void LateUpdate()
+    {
+        Vector3 newposition = target.transform.position + cameraOffset;
+        //transform.position = Vector3.Slerp(transform.position, newposition,smoothFactor);
+        transform.position = newposition;
```

Comment:
Hint: Every time you use position consider whether you should use localPosition instead. What is the implicit logic you have in this class? I think you are fine here, but wanted to point it out.